uniflow/flow/config.py

```diff
@@ -241,11 +242,184 @@ class RaterClassificationOpenAIGPT4Config(RaterConfig):
+                    generated_answer="Yes. Early computers were built to perform a series of single tasks, similar to a calculator.",
+                    explanation="The generated answer is better as it correctly captures the essence of the early computers' functionality, which was to perform single tasks akin to calculators.",
+                    label="accept",
+                ),
+                Context(
+                    context="Operating systems did not exist in their modern and more complex forms until the early 1960s. Hardware features were added, that enabled use of runtime libraries, interrupts, and parallel processing.",
+                    question="When did operating systems start to resemble their modern forms?",
+                    grounding_answer="Operating systems started to resemble their modern forms in the early 1960s.",
+                    generated_answer="Modern and more complex forms of operating systems began to emerge in the early 1960s.",
+                    explanation="Both answers are equally good as they accurately pinpoint the early 1960s as the period when modern operating systems began to develop.",
+                    label="equivalent",
+                ),
+                Context(
+                    context="Operating systems did not exist in their modern and more complex forms until the early 1960s. Hardware features were added, that enabled use of runtime libraries, interrupts, and parallel processing.",
+                    question="What features were added to hardware in the 1960s?",
+                    grounding_answer="Hardware in the 1960s saw the addition of features like runtime libraries and parallel processing.",
+                    generated_answer="The 1960s saw the addition of input output control and compatible timesharing capabilities in hardware.",
+                    explanation="The generated answer is worse because it inaccurately suggests the addition of capabilities of hardware in 1960s which is not supported by the context.",
+                    label="reject",
+                ),
+                Context(
+                    context="Operating systems did not exist in their modern and more complex forms until the early 1960s. When personal computers became popular in the 1980s, operating systems were made for them similar in concept to those used on larger computers.",
+                    question="When did operating systems in personal computer were similar to those used on larger computers?",
+                    grounding_answer="In 1980s, as personal computers became popular.",
+                    generated_answer="In the early 1960s, as operating system became more complex.",
+                    explanation="The generated answer is much worse as it incorrectly states the early 1960s as the period of popularity for personal computers, contradicting the context which indicates the 1980s.",
+                    label="strong reject",
```

Comment:
This is a very nice example. However, this long token length will cause a high cost. I would suggest to write an example with only 'accept`, `equivalent`, and `reject` to reduce cost.